src/scripts/main.js

```diff
@@ -1,3 +1,381 @@
 'use strict';
 
-// write code here
+const tableHeaders = [...document.querySelectorAll('thead th')];
+let tableRows = [...document.querySelectorAll('tbody tr')];
```

Comment:
Similar to the previous line, `document.querySelectorAll` is incorrectly used with square brackets. It should be `document.querySelectorAll('tbody tr')`.